lib/src/rules/length.ts

```diff
@@ -0,0 +1,29 @@
+export interface LengthParams {
+  length: number;
+}
+
+export function parseLengthParams(customParams: LengthParams, errorMessage: string) {
+  const length = customParams.length === null ? NaN : Number(customParams.length);
```

Comment:
It is declared as a number in TypeScript but when it is transpiled there is no type checking in runtime.